dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -610,7 +611,18 @@ export const LiveLayout = ({ CAPI, NAV, format, palette }: Props) => {
 								/>
 							</GridItem>
 							<GridItem area="lastupdated">
-								<></>
+								<Hide until="desktop">
+									{CAPI.blocks.length &&
+										CAPI.blocks[0].blockLastUpdated && (
```

Comment:
Is this the same as `content.fields.lastModified`?

Reply:
When we load new blocks we don't get access to the full CAPI body, just the new blocks, so we won't have access to content.fields.lastModified. I think perhaps we should keep using blockLastUpdated because of that.